contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryBase2_3.sol

```diff
@@ -103,6 +103,9 @@ abstract contract AutomationRegistryBase2_3 is ConfirmedOwner {
   mapping(uint256 => bytes) internal s_upkeepOffchainConfig; // general config set by users for each upkeep
   mapping(uint256 => bytes) internal s_upkeepPrivilegeConfig; // general config set by an administrative role for an upkeep
   mapping(address => bytes) internal s_adminPrivilegeConfig; // general config set by an administrative role for an admin
+  // billing
+  mapping(address billingToken => BillingConfig billingConfig) internal s_billingConfigs; // billing configurations for different tokens
+  address[] internal s_billingTokens; // list of billing tokens
```

Comment:
one more comment - can we make these `IERC20`s instead of `address`s? you can get that from `vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol`

Reply:
Sure, that makes more sense. I guess we should use IERC20 for both the mapping and the list

Reply:
yeah pretty much everywhere - including in the functions like `getBillingConfig()` and `setConfig()`, etc

Reply:
I updated the underlying data structures and associated APIs. I didnt touch the setConfigTypeSafe signature tho, let me know if you think that need to use IERC20 as well.

Reply:
yeah I think make that `IERC20` also!